### PR TITLE
Pal 1055 include jre jdk in tags

### PR DESCRIPTION
**Pull request by m78233**

### CODE REVIEW

File: .hadolint.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@
 #and add in this format: - DL3000
 ignored:
 - DL3006
+- DL3018
+- DL3008
+
```
Comment (pd104923) [inline .hadolint.yaml:22]: I don't think we need to have an extra line here.

**Review by pd104923 — CHANGES_REQUESTED** [review]
1 minor suggestions

File: .hadolint.yaml (hunk 1 above)
Comment (m78233) [reply .hadolint.yaml:22]: removed extra line

**Review by pd104923 — APPROVED** [review]

File: deployment/Dockerfile [hunk 2]
```diff
@@ -13,12 +13,13 @@
...
 FROM alpine:3.12.0@sha256:90baa0922fe90624b05cb5766fa5da4e337921656c2f8e2b13bd3c052a0baac1
+
 VOLUME /tmp
 
 ARG K8_VERSION=v1.15.3
 
-RUN apk --no-cache add curl=7.76.1-r0
-RUN apk --no-cache add bash=5.0.17-r0
+RUN apk --no-cache add curl
+RUN apk --no-cache add bash
```
Comment (developer01189998819991197253) [inline deployment/Dockerfile:22]: looking at the hadolint rule: https://github.com/hadolint/hadolint/wiki/DL3018 
We can specify a partial version using `=~5.0.17`
Update the curl and bash versions in the dockerfile to use the partial versions and update the hadolint yaml rules appropriatley

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

File: deployment/Dockerfile (hunk 2 above)
Comment (m78233) [reply deployment/Dockerfile:22]: Interesting about the partial version, I did not know about that.  I re-visited the ticket and it was specific to ask that we use the latest version of curl and bash.   We were even asked to add a hadolint.yaml to suppress any complaints.    I took all of this to mean that in this instance we don't care about the version.  As I'm not sure which way this should go, I have left it as is.

**Review by dev930018 — APPROVED** [review]

**Pull request merged**